containers/ddev-xhgui/Dockerfile

```diff
@@ -1,4 +1,20 @@
-FROM xhgui/xhgui:0.21 AS ddev-xhgui
+FROM xhgui/xhgui:0.21 AS xhgui-docker
+
+FROM scratch AS ddev-xhgui
```

Comment:
I wasn't suggesting to re-do this :) The other way was fine too :)

Reply:
We don't need to edit this line in the future with `scratch` (when `xhgui/xhgui` changes the base image).